pkg/sql/sem/builtins/builtins.go

```diff
@@ -69,6 +69,9 @@ var (
 		"input value must be <= %d (maximum Unicode code point)", utf8.MaxRune)
 	errStringTooLarge = pgerror.Newf(pgcode.ProgramLimitExceeded,
 		"requested length too large, exceeds %s", humanizeutil.IBytes(maxAllocatedStringSize))
+	// SequenceNameArg represents the name of sequence (string) arguments in
```

Comment:
How about `SequenceNameArg represents the name of an argument to a builtin that is a sequence name represented as a string`?